src/test/java/duke/commands/DeleteTaskCommandTest.java

```diff
@@ -0,0 +1,40 @@
+package duke.commands;
+
+import duke.exceptions.InvalidTaskNumberException;
+import duke.exceptions.InvalidTimeStampException;
+import duke.utils.TaskList;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+public class DeleteTaskCommandTest {
+
+    @Test
+    public void testValidDeleteCommand() {
+        String expected = "TO DELETE: index 10";
+        Command actual = new DeleteTaskCommand("10");
+        assertEquals(
+                expected,
+                actual.toString()
+        );
+    }
+
+    @Test
+    public void test0Index() {
```

Comment:
Maybe this can be changed to follow the featureUnderTest_testScenario_expectedBehavior() format? Something like deleteCommand_zeroIndex_exceptionThrown()